docs/integrations/nbext.md

```diff
@@ -1,47 +1,53 @@
 
 ## Using Jovian Jupyter Extension
 
-<img src="https://i.imgur.com/RdBnJYy.png" class="screenshot" alt="jupyter extension toolbar button">
+<img src="https://i.imgur.com/44QTHP6.png" class="screenshot" alt="jupyter extension toolbar button">
 
 Once you have successfully installed jovian, a new button `Commit` will appear on the tool bar. When using `Commit` button for first time you'll be asked to provide an API key.
 
 <img src="https://i.imgur.com/oNSowtY.png" class="screenshot" alt="jovian api key prompt">
 
 You can get the API key at [Jovian](https://jovian.ml?utm_source=docs). Once you log in, just click on `API key` button, and the key will be copied to the clipboard.
 
-<img src="https://i.imgur.com/taLLUVd.png" class="screenshot" alt="jovian api key copy button" >
+<img src="https://i.imgur.com/taLLUVd.png" class="screenshot" alt="jovian api key copy button">
 
 #### Valid API key
 
 If the key is valid you will be notified with the following alert.
 
-<img src="https://i.imgur.com/UHvSihx.png" class="screenshot" alt="valid api key prompt" >
+<img src="https://i.imgur.com/UHvSihx.png" class="screenshot" alt="valid api key prompt">
 
 #### Error with API key
 
 If the entered API key is invalid you will get following error.
 
-<img src="https://i.imgur.com/9WaVkTR.png" class="screenshot" alt="api key error prompt" >
+<img src="https://i.imgur.com/9WaVkTR.png" class="screenshot" alt="api key error prompt">
 
 #### Successful Commit
 
 Once the API key has been validated, you can start committing to [Jovian](https://jovian.ml?utm_source=docs) by clicking `Commit` button. Once the Notebook has been committed successfully you will get the confirmation message with the link where the Jupyter Notebook has been uploaded to, you can use the copy button to get the link to the share the notebook.
 
-<img src="https://i.imgur.com/4GoqzER.png" class="screenshot" alt="jovian commit success prompt" >
+<img src="https://i.imgur.com/4GoqzER.png" class="screenshot" alt="jovian commit success prompt">
 
-## Commit Dropdown to commit with more options
+## Commit with more options
 
-This makes use of [jovian.commit's](../jvn/commit) parameters to enable the user to commit with preferences like secret notebook, new notebook project, to add output and artifact files .....
+This makes use of [jovian.commit's](https://jovian-py.readthedocs.io/en/latest/jvn/commit.html) parameters to enable the user to commit with preferences like private notebook, new notebook project, to add outputs and files .....
 
-<img src="https://i.imgur.com/maHhYY2.png" class="screenshot" alt="jovian commit options jupyter extension" >
+Firstly, click the dropdown menu
 
-By default the parameters are derived from [jovian.commit](../jvn/commit), once the user changes a certain parameter they are retained.
+<img src="https://i.imgur.com/C4trCHn.png" class="screenshot" alt="jovian dropdown menu">
```

Comment:
Can you change these assets, to make it smaller.
You can minimize the browser and then take a screenshot, else there is a lot of information in the bottom which is not relevant to the component you want to highlight.

Also can you use oval/circle highlighter to be consistent.

Reply:
Sure, do i want me only screen shot the tool bar infos such as 
![Screenshot from 2020-03-02 23-47-23](https://user-images.githubusercontent.com/38867924/75753723-35026300-5ce0-11ea-840e-b04383329e38.png)?

Reply:
Have the whole window to retain the context, but resize the window.

Reply:
![image](https://user-images.githubusercontent.com/24864161/75754282-759ac480-5d52-11ea-8bd6-cc3e51a9eab0.png)
Something like this

Reply:
okay, I will try to retake screenshot first, and then you can check to see if I need change